### ISSUE TITLE
Beta: IDE crashes at start if gms.yoyogames.com can't be resolved

### ISSUE THREAD

**Issue by stravee** [YoYoGames/GameMaker-Bugs#7034]
### Description

![beta ide crash after start 2024-07-31 172543](https://github.com/user-attachments/assets/0b53959b-2999-4cbd-9f57-f13b1b60c461)

I'm attaching ui.log

[ui.log](https://github.com/user-attachments/files/16444055/ui.log)

### After this line it crashes:
DoUpdateDownload exception System.Net.WebException: No such host is known. (gms.yoyogames.com:443)
 ---> System.Net.Http.HttpRequestException: No such host is known. (gms.yoyogames.com:443)
 ---> System.Net.Sockets.SocketException (11001): No such host is known.

### Like this:
[17:25:21:451(2a26)] Unhandled Exception: System.Net.WebException
[17:25:21:451(2a26)] Exception message: No such host is known. (gms.yoyogames.com:443)
[17:25:21:452(2a26)] Exception stack:    at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)

### Steps To Reproduce

...

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.800 (Betas)

### Which operating system(s) are you seeing the problem on?

Windows 11

### Are you running GameMaker from inside your Steam library?

No

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by YYDan** [comment]
Yes, the ui.log has a series of errors right at the end for not being able to reach our servers and check the RSS feeds. Will investigate this, as we're currently investigating other problems with these feeds not being reliable when accessed by the IDE. 

Thanks!

**Comment by YYDan** [comment]
I will point out, though, that your log has a lot of permissions errors on almost every IDE session and _looks_ like it crashes often on startup because you're blocking/firewalling the MQTT server, plus a see a lot of lines about various files being locked when GM is trying to access them.

Slightly aside from this crash (although it might well help stop the crash from happening again also) you might want to review our [Permissions FAQ](https://help.gamemaker.io/hc/en-us/articles/360022953052) and follow the steps there. Note that we always link to this guide on the release notes page, as many users need to review their protection settings every time they install a new version of GameMaker, so this may be something you need to do also in future.

**Comment by stravee** [comment]
I haven't had an IDE crash in a long while and also no related issues working with GM (beta or stable), so I'm not sure about those locked files, but I'll look into it.

Thanks for your feedback!
